tenantvariables/pom.xml

```diff
@@ -89,6 +89,32 @@
 			</exclusions>
 		</dependency>
 
+
```

Comment:
Better put the versions of the depedencies in the central file, so that they can centrally be managed.

Reply:
Yes, I forgot to do that at the end.

Reply:
done!